ResearchKit/Common/ORKFormStep.m

```diff
@@ -81,6 +81,17 @@ - (void)validateParameters {
     for (ORKFormItem *item in _formItems) {
         [item.answerFormat validateParameters];
     }
+    
+    [self validateIdentifiersUnique];
+}
+
+- (void)validateIdentifiersUnique {
+    NSArray *uniqueIdentifiers = [_formItems valueForKeyPath:@"@distinctUnionOfObjects.identifier"];
+    BOOL itemsHaveNonUniqueIdentifiers = _formItems.count != uniqueIdentifiers.count;
+    
+    if (itemsHaveNonUniqueIdentifiers) {
+        @throw [NSException exceptionWithName:NSGenericException reason:@"All form items should have unique identifier" userInfo:nil];
```

Comment:
Should we use `[_formItems count]` (90 uses through the project) or `_formItems.count` (53 uses)? Would be good te standardize on this as well.